src/components/dashboard/main-content/course-enrollments/tests/CourseEnrollments.test.jsx

```diff
@@ -60,28 +58,39 @@ function renderEnrollmentsComponent(initialProps) {
     </Provider>,
   );
 }
+describe('Course enrollements', () => {
+  beforeEach(() => {
+    updateCourseCompleteStatusRequest.mockImplementation(() => {});
+  });
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  test('loads enrollments component', () => {
+    renderEnrollmentsComponent(initProps);
+    expect(screen.getByText('My courses in progress')).toBeInTheDocument();
+    expect(screen.getByText('Completed courses')).toBeInTheDocument();
+    expect(screen.getByText('Courses saved for later')).toBeInTheDocument();
+    expect(screen.getAllByText('edX Demonstration Course').length).toBeGreaterThanOrEqual(1);
+  });
 
-test('loads enrollments component', () => {
-  renderEnrollmentsComponent(initProps);
-  expect(screen.getByText('My courses in progress')).toBeInTheDocument();
-  expect(screen.getByText('Courses saved for later')).toBeInTheDocument();
-  expect(screen.getAllByText('edX Demonstration Course').length).toBeGreaterThanOrEqual(1);
-});
+  it('generates course status update on move to in progress action', () => {
+    renderEnrollmentsComponent({ ...initProps });
```

Comment:
does this vs. L69 both work the same? the one above uses just 'initProps', will be good to doublecheck ..

Reply:
guessing it is ok/better to pass just initProps here..just like L69

Reply:
sorry to be clear: let's go with just initProps here? the same results I think, but just  think that spreading here is not adding anything